### DISCUSSION TITLE
Malwarebytes detecting Ghelper 1.70.0 as Malware

### DISCUSSION THREAD

**Discussion by Reetrox** [seerge/g-helper#2608] [Q&A] [answered]
Has anyone else been having this issue? It just started happening to me today.

**Comment by seerge** [comment, marked as answer]
> [**Windows Defender marks app as malware / virus**](https://github.com/seerge/g-helper/wiki/FAQ#windows-defender-marks-app-as-malware--virus)
> False positives from Windows Defender (or any other similar system that uses machine learning for detection) is possible as the application is not digitally signed with a certificate. You can always download a version below or compile the app by yourself. All application sources are open and can be monitored from A to Z :)

https://www.virustotal.com/gui/url/64bc4a9446705d29571219681966ddf84107600ce4a6770461bc1396c2387604
